server/src/main/java/io/seata/server/cluster/raft/RaftServerFactory.java

```diff
@@ -155,6 +157,20 @@ public void start() {
         }
     }
 
+    @Override
+    public void destroy() {
+        this.close();
+    }
+
+    @Override
+    public void close() {
```

Comment:
The general purpose of the Java Factory class is to create and return objects of other classes, not to control behavior. If rpcServer is part of RaftServer, then it is more appropriate to control behavior in RaftServer.

Reply:
> The general purpose of the Java Factory class is to create and return objects of other classes, not to control behavior. If rpcServer is part of RaftServer, then it is more appropriate to control behavior in RaftServer.

A single rpc server can be used by multiple raft servers, so all raft servers should be controlled in the raft server factory, as well as their behavior when destroyed.